docker/docker-compose.yml

```diff
@@ -305,11 +321,12 @@ services:
     depends_on:
       - things
       - nats
+      - rabbitmq
```

Comment:
Adding both Nats and RabbitMQ as dependencies is suboptimal. The one should choose between them, rather than always having them running together. Consider using multiple compose files (one for broker and the other for the Mainflux services), updating Makefile, or checking if there are options to do it with a single compose file.